assets/component-slider.css

```diff
@@ -90,6 +96,13 @@ slider-component.slider-component-full-width {
     margin-bottom: 1rem;
   }
 
+  .slider--tablet:after {
```

Comment:
I even know what we're doing here (kind of) and these rules confuse me. Can we add a comment above this selector that explains what this is for?

Reply:
Added some comments, let me know if they're clear enough 🦖